src/App.css

```diff
@@ -33,5 +36,103 @@ body {
   padding: 2px;
 }
 h1 {
-  font-size: 20px;
+  font-size: 20px; 
+  font-weight: 600;
+  border-bottom: 2px solid #090707;
+}
+.friends {
+  width: auto;  
+  height: 100px;
```

Comment:
Giving height property is discouraged, it may lead to unexpected bugs. Let the height be decided by margin and padding

Reply:
removed